doc/lsp.txt

```diff
@@ -282,6 +282,17 @@ diagLineHL		Highlight used for diagnostic line.
 echoSignature		In insert mode, echo the current symbol signature
 			instead of showing it in a popup. By default this is
 			set to false.
+completionMatcher	Enable fuzzy or incase sensitive completion for language
+			servers that replies with a full list of completion
+			items.  Some language servers does completion filtering
+			in the server, while other relies on the client to do
+			the filtering.
+
+			This option only works for language servers that expect
+			the client to filter the completin items.
+
+			Can be either `fuzzy`, `icase` or `case`, default is
+			`case`.
```

Comment:
Currently "case" is the default. Which isn't backward compatible as ignored case completion have been added a week ago, and "completed" in #167

Reply:
Is `case` a good default? I'd find it a bit annoying with languages that use camelCase like Java

Reply:
I don't know.

I find it annoying that `Hello` and `hello` will both be suggested when I type: `He`.

Vim's built-in completion `:help ins-completion` is case sensitive, with the twist that can set `:h 'infercase`.

Reply:
I'm quite new to Vim, so if Vim's general default is case sensitive then it makes to have that default here too

Reply:
Either is fine I think. My personal favorite is `case` as that's how I would configure it myself.